### PR TITLE
Add ESLint and JSCS

### LINKED ISSUE #29: Add jscs
Use jscs-preset-gulp

### LINKED ISSUE #30: Explicit eslint usage
To bring inline with gulp repos. Use eslint-config-gulp and update code style

### LINKED ISSUE #31: Update repo location references
In package.json, readme, etc

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #29 
Fixes #30 
Fixes #31

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,14 +1,7 @@
 {
-  "rules": {
-    "semi": [2, "always"],
-    "quotes": [2, "single"],
-    "strict": [2, "global"],
-    "no-underscore-dangle": 0,
-    "no-use-before-define": 0
-  },
+  "extends": "gulp",
   "env": {
```
Comment (phated) [inline .eslintrc]: these can be removed

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,8 @@
-sudo: false
 language: node_js
+
+sudo: false
```
Comment (phated) [inline .travis.yml]: I prefer sudo: false to be at the top

File: package.json [hunk 3]
```diff
@@ -19,6 +19,8 @@
     "LICENSE"
   ],
   "scripts": {
+    "lint": "eslint . && jscs *.js test/",
+    "pretest": "npm run lint",
     "test": "lab -cvL"
```
Comment (phated) [inline package.json]: remove linting from the lab command

**Comment by pdehaan** [comment]
Done.
Not sure if Travis-CI needs to be re-enabled for this org/repo.

**Comment by phated** [comment]
I've tried toggling it.  Can you push something here and see if it is fixed?

**Comment by pdehaan** [comment]
Done.

**Pull request merged**